docs/pages/updating.md

```diff
@@ -0,0 +1,109 @@
+# Updating
+
+### Version 6.0.0
+
+#### iOS
+
+In your project Podfile add support for static dependency linking. This is required to support the new Promises subdependency in the iOS swift conversion.
+
+Add `use_frameworks! :linkage => :static` just under `platform :ios` in your ios project Podfile.
+
+[See the example ios project for reference](examples/basic/ios/Podfile#L5)
+
+### Version 5.0.0
+
+Probably you want to update your gradle version:
+#### gradle-wrapper.properties
+```diff
+- distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-all.zip
++ distributionUrl=https\://services.gradle.org/distributions/gradle-5.1.1-all.zip
+```
+
+#### **android/app/build.gradle**
+
+From version >= 5.0.0, you have to apply this changes:
+
+```diff
+dependencies {
+   ...
+    compile project(':react-native-video')
++   implementation "androidx.appcompat:appcompat:1.0.0"
+-   implementation "com.android.support:appcompat-v7:${rootProject.ext.supportLibVersion}"
+
+}
+```
+
+#### **android/gradle.properties**
+
+Migrating to AndroidX (needs version >= 5.0.0):
+
+```groovy
+android.useAndroidX=true
+android.enableJetifier=true
+```
+
+### Version 4.0.0
```

Comment:
don't you think we can remove this update part ?
I agree to remove it

Reply:
Done ✅